src/fides/api/service/privacy_request/request_runner_service.py

```diff
@@ -416,6 +402,9 @@ def run_privacy_request(
                 request_checkpoint=CurrentStep.upload_access,
                 from_checkpoint=resume_step,
             ):
+                privacy_request.cache_failed_checkpoint_details(
+                    CurrentStep.upload_access
+                )
```

Comment:
Now I am more using "cache_failed_checkpoint_details" to update every time a checkpoint is hit, rather than just some selected locations when a Privacy Request fails to have a better understanding of the latest checkpoint even when a Privacy Request fails in unexpected ways.